assets/component-search.css

```diff
@@ -1,32 +1,70 @@
 .search__input.field__input {
-  padding-right: 5rem;
+  padding-right: 9.8rem;
 }
 
 .search__button {
```

Comment:
After clearing the search term, the search results dont stop and keep loading:

https://screenshot.click/06-03-ykpwy-cj92h.mp4

Is this expected? Shouldnt it be paused?

Reply:
Nice suggestion!